Good-Bids/goodbids
# [#470] My account polish

# Summary

This styles all the pages in the My Account section by rounding the table corners and updating the stats blocks. 

There are also edits to the following pages.
- Bids
   - DONATION HISTORY header above table
- Free Bids
  - Stats
  - Copy Referral section
  - FREE BIDS EARNED header above table
- Rewards
  - Stats (already there just had the wrong file name must have got ten lost in a merge)
  - CLAIMED REWARDS header above table
- Downloads (Question in the ticket)
- Payment Methods
  -  Page Title
  - Helper copy
- Account Details
  - Add Page Title

I had adjusted the base border radius to be fixed vs using the WP spacing which was fluid. But I noticed that a lot of our rounded edges are using the old size when they should be using `rounded-sm` to match the design. So I updated all the sections that needed to move to `rounded-sm`.

## Issues

* #470

## Testing Instructions

1. Create a new site (existing sites have the old WooCommerce templates cached)
2. Go to the My Account on the front end.
3. Click through all the navigation link.
4. Make sure everything looks correct. 

## Screenshots


| Header | Header |
|--------|--------|
| <img width="1112" alt="Screenshot 2024-02-26 at 4 51 30 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/d12d667f-6458-4846-b045-d8e1b4bfe59c"> | <img width="1119" alt="Screenshot 2024-02-26 at 4 52 25 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/8c56a02e-63d9-4985-ac15-127bc1569730"> |
| <img width="1126" alt="Screenshot 2024-02-26 at 7 17 13 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/ab2ca7b7-2f0b-4246-a60f-f332df1d629e"> | <img width="1080" alt="Screenshot 2024-02-26 at 4 52 34 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/737b7230-f353-4ba2-95dc-fadbdff9e900"> |
| <img width="1116" alt="Screenshot 2024-02-26 at 7 18 55 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/1595e664-38d3-49ac-a121-94fe1b73ba5c"> | <img width="992" alt="Screenshot 2024-02-26 at 7 20 24 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/e857c690-4fa9-44d6-a974-6005b438941d">|
